### PR TITLE
#7 Estimate project size

### PR DESCRIPTION
**Pull request by super132**
For task #7, provide estimate of the component.

### CODE REVIEW

**Comment by super132** [comment]
@dmarkov please review this PR. Thanks.

**Comment by dmarkov** [comment]
@super132 Thanks, let me find someone who can review this pull request

**Comment by dmarkov** [comment]
@pinaf please review this one

**Comment by dmarkov** [comment]
> @dmarkov please review this PR. Thanks.

@super132 thank you, I will check it

File: est/jcabi-slf4j.est [hunk 1]
```diff
@@ -0,0 +1,18 @@
+date: 27-05-2015
+author: Jason Wong
+method: champions.pert
+scope:
+  1: jcabi-maven-slf4j maven plugin project scaffolding
+  2: Implementation of SLF4J maven adpator
```
Comment (pinaf) [inline est/jcabi-slf4j.est]: @super132 typo here: `adaptor`

**Comment by pinaf** [comment]
@super132 1 minor comment above

**Comment by super132** [comment]
@pinaf typo is fixed. Would you please check? Thanks.

**Comment by pinaf** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by dmarkov** [comment]
@pinaf Thanks so much! Your account was topped up for **17 mins** (transaction ID is `AP-3JA819586U2755014`, the task took 140 hours and 23 mins). extra minutes for review comments (c=2). added +17 to your rating, now it is equal to [+7737](http://www.netbout.com/b/35128?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**